src/constants.py

```diff
@@ -1,73 +0,0 @@
-import re
-
-file_identifier_regex_map = {
```

Comment:
While it is a good idea to declare the funds and filename formats, there is no need to call `re.compile` on the regular expressions here.

Reply:
In addition, by keeping everything as strings, you could use a declarative format such as JSON or YAML.